docs/configuration.md

```diff
@@ -275,6 +275,17 @@ There are a few exceptions to these rules:
 1. Any config-file-supported settings that are provided on the command-line (e.g., via
     `--select`) will override the settings in _every_ resolved configuration file.
 
+When no discovered configuration specifies a [`target-version`](settings.md#target-version), Ruff will attempt to fallback to the minimum version compatible with the `requires-python` field in a nearby `pyproject.toml`.
+If this attempt fails, Ruff will use the default `target-version`.
```

Comment:
I think I'd omit this. It breaks the reading flow and is implicit by *attempt to fallback*